src/hashstore/hashstore.py

```diff
@@ -76,871 +13,141 @@ def version():
-            s_path = self.open(entity, ab_id)
-            s_content = s_path.read().decode("utf-8").split("\x00", 1)
-            s_path.close()
-            sysmeta = s_content[1]
-        else:
-            raise ValueError(f"No sysmeta found for pid: {pid}")
-        return sysmeta
-
-    def delete_object(self, pid):
-        if pid is None or pid.replace(" ", "") == "":
-            raise ValueError(f"Pid cannot be None or empty, pid: {pid}")
-
-        entity = "objects"
-        ab_id = self.get_sha256_hex_digest(pid)
-        self.delete(entity, ab_id)
-        return True
-
-    def delete_sysmeta(self, pid):
-        if pid is None or pid.replace(" ", "") == "":
-            raise ValueError(f"Pid cannot be None or empty, pid: {pid}")
-
-        entity = "sysmeta"
-        ab_id = self.get_sha256_hex_digest(pid)
-        self.delete(entity, ab_id)
-        return True
-
-    def get_hex_digest(self, pid, algorithm):
-        if pid is None or pid.replace(" ", "") == "":
-            raise ValueError(f"Pid cannot be None or empty, pid: {pid}")
-        if algorithm is None or algorithm.replace(" ", "") == "":
-            raise ValueError(f"Algorithm cannot be None or empty, pid: {pid}")
-
-        entity = "objects"
-        algorithm = self.clean_algorithm(algorithm)
-        ab_id = self.get_sha256_hex_digest(pid)
-        if not self.exists(entity, ab_id):
-            raise ValueError(f"No object found for pid: {pid}")
-        c_stream = self.open(entity, ab_id)
-        hex_digest = self.computehash(c_stream, algorithm=algorithm)
-        return hex_digest
-
-    # FileHashStore Core Methods
-
-    def put_object(
-        self,
-        pid,
-        file,
-        extension=None,
-        additional_algorithm=None,
-        checksum=None,
-        checksum_algorithm=None,
-    ):
-        """Store contents of `file` on disk using the hash of the given pid
+        Upon successful storage of sysmeta, the method returns a String that
+        represents the file's permanent address, and similarly to 'store_object', this
+        permanent address is determined by calculating the SHA-256 hex digest of the
+        provided pid. Finally, sysmeta are stored in parallel to objects in the
+        `/store_directory/sysmeta/` directory.
 
         Args:
-            pid (string): authority-based identifier
-            file (mixed): Readable object or path to file.
-            extension (str, optional): Optional extension to append to file
-                when saving.
-            additional_algorithm (str, optional): Optional algorithm value to include
-                when returning hex digests.
-            checksum (str, optional): Optional checksum to validate object
-                against hex digest before moving to permanent location.
-            checksum_algorithm (str, optional): Algorithm value of given checksum
+            pid (string): Authority-based identifier. \n
```

Comment:
`\n` typo?

Reply:
Not a typo - I added the new line there so that the Args section in the docstrings when you hover over the method split up nicely. However... it might have had something to do with my VSCode not being updated - as I removed in 'hashstore.py' and it looks fine now.

However... in `filehashstore.py` it still doesn't preview nicely if I remove `\n`. Leaving the new lines in `filehashstore.py` for now.

Example without `\n`:
<img width="626" alt="image" src="https://github.com/DataONEorg/hashstore/assets/12792837/30ad4f98-5d98-47c1-99b5-92285063fb12">